tutorials/blitz/7_captum_explainability.py

```diff
@@ -0,0 +1,203 @@
+"""
+Explaining Graph Neural Network with Captum
+===========================================
+
+By the end of this tutorial, you will be able to:
+
+- Understand the basic concepts of GNN explainability
+- Use Captum to explain the node classification task
+- Find the most important nodes and edges to the model
+- Visualize the node-centered subgraphs
+
+(Time estimate: 20 minutes)
+
+"""
+
+######################################################################
+# Overviews of GNN Explainability
+# -------------------------------
+# In recent years, research on the interpretability of deep learning models has
+# made significant progress. Compared to cv and nlp domains, there is less
+# research and application of graph model interpretability, yet it is the key
+# to understanding deep graph neural networks. Generally, GNN explanation
+# research often starts with the following tasks: ** Which input edges are more
+# important? Which input nodes are more important? **
+
+# Node-centered subgraphs play a critical role in analyzing GNNs. The k-hop
+# subgraph of a node fully determines the information a k-layer GNN exploits
+# to generate its final node representation. Many GNN explanation methods
+# provide explanations by extracting a subgraph and assigning importance
+# weights to the nodes and edges of it. We will visualize node-centered
+# weighted subgraphs through DGL's built-in functions. This is beneficial
+# for debugging and understanding GNNs and GNN explanation methods.
+
+# For this demonstration, we will use IntegratedGradients from `Captum <https://github.com/pytorch/captum>`__ to
+# explain the predictions of a graph convolutional network (GCN).
+# Specifically, we try to find the most important nodes and edges to the model in node classification.
+# Captum is a model interpretability and understanding library for PyTorch. You can install it with
+
+# .. code:: bash
+#
+# pip install captum
+#
+
+
+######################################################################
+# Loading Cora Dataset
+# --------------------
+#
+# First, we load DGL’s built-in Cora dataset and retrieve its graph structure, node labels (classes) and the number of node classes.
+# 
+
+# Install and import required packages.
+import dgl
+import dgl.data
+# The Cora dataset used in this tutorial only consists of one single graph.
+dataset = dgl.data.CoraGraphDataset()
+g = dataset[0]
+
+######################################################################
+# Define the model
+# ----------------
+# Then, we will build a two-layer Graph Convolutional Network (GCN).
+# Each layer computes new node representations by aggregating neighbor information.
+# What's more, we use GraphConv which supports ``edge_weight`` as a parameter to calculate
+# the importance of the edge in the edge explainability task.
+# 
+
+from dgl.nn import GraphConv
+# Define a class for GCN
+class GCN(nn.Module):
+    def __init__(self, in_feats, h_feats, num_classes):
```

Comment:
```suggestion
    def __init__(self, in_feats, num_classes, h_feats=16):
```

You can then remove `16` in `GCN(g.ndata['feat'].shape[1], 16, dataset.num_classes)`.